README.md

```diff
@@ -69,7 +69,27 @@ dependencies {
 
 ### Usage
 
-// TODO
+Register `Search Manager`
+```java
+SearchManager searchManager = new SearchManager();
+```
+
+Register sources
+```java
+var spotify = new SpotifySourceManager(clientId, clientSecret, spDc, countryCode, () -> audioPlayerManager, DefaultMirroringAudioTrackResolver)
+var youtube = new YoutubeSearchManager(() -> audioPlayerManager, region);
+
+searchManager.registerSearchManager(spotify);
+searchManager.registerSearchManager(youtube);
+```
+
+Basic usage
+```java
+AudioSearchResult searchResult = searchManager.loadSearch(query, types);
+```
+
+- query: String
+- types: Set of `AudioSearchResult.Type`
```

Comment:
```suggestion
Basic usage
```java
var query = "ytmsearch:bla bla";
var types = List.of(
	AudioSearchResult.Type.TRACK,
	AudioSearchResult.Type.PLAYLIST,
	AudioSearchResult.Type.ALBUM,
	AudioSearchResult.Type.ARTIST,
	AudioSearchResult.Type.TEXT
);

AudioSearchResult searchResult = searchManager.loadSearch(query, types);
```